leo/cli/commands/build.rs

```diff
@@ -219,5 +213,9 @@ fn compile_leo_file<N: Network>(
         .map_err(CliError::failed_to_load_instructions)?;
 
     tracing::info!("✅ Compiled '{program_name}.aleo' into Aleo instructions");
+    if enable_dce {
```

Comment:
Nit. Could we move this before the `Compiled .* into Aleo instructions`? And then also indent the two messages?